release-notes/7.0/README.md

```diff
@@ -1,7 +1,6 @@
 # .NET 7
 
-[.NET 7](https://devblogs.microsoft.com/dotnet/announcing-dotnet-7-preview-2/) will be a [current release](../../release-policies.md) and will be supported for 18 months, from November 2022 to May 14, 2024. It is [supported by Microsoft](../../microsoft-support.md) on [multiple operating systems](supported-os.md).
-
+[.NET 7](https://devblogs.microsoft.com/dotnet/announcing-dotnet-7-preview-4/) is an [STS release](../../release-policies.md) and will be supported for 18 months, from November 8th, 2022 to May 14th, 2024. It is [supported by Microsoft](../../microsoft-support.md) on [multiple operating systems](supported-os.md).

```

Comment:
```suggestion
[.NET 7](https://devblogs.microsoft.com/dotnet/announcing-dotnet-7-preview-4/) is a [standard-support release](../../release-policies.md) and will be supported for 18 months, from November 8th, 2022 to May 14th, 2024. It is [supported by Microsoft](../../microsoft-support.md) on [multiple operating systems](supported-os.md).

```

Reply:
Note that Windows also does caps for LTSC -> https://learn.microsoft.com/en-us/windows/whats-new/ltsc/

Reply:
They are wrong too 😂